modules/integration/tests-integration/tests-backend/src/test/resources/org/wso2/identity/integration/test/rest/api/server/idp/v1/add-idp-oidc-standard-based.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "OIDC IdP",
+  "alias": "",
+  "description": "Authenticate users with Enterprise OIDC connections.",
+  "image": "assets/images/logos/enterprise.svg",
+  "isPrimary": false,
+  "roles": {
+    "mappings": [],
+    "outboundProvisioningRoles": []
+  },
+  "certificate": {
+    "jwksUri": "https://test.com/jwks",
+    "certificates": [
+      ""
+    ]
+  },
+  "claims": {
+    "userIdClaim": {
+      "uri": ""
+    },
+    "provisioningClaims": [],
+    "roleClaim": {
+      "uri": ""
+    }
+  },
+  "federatedAuthenticators": {
+    "defaultAuthenticatorId": "T3BlbklEQ29ubmVjdEF1dGhlbnRpY2F0b3I",
+    "authenticators": [
+      {
+        "isEnabled": true,
+        "authenticatorId": "T3BlbklEQ29ubmVjdEF1dGhlbnRpY2F0b3I",
+        "properties": [
+          {
+            "key": "ClientId",
+            "value": "abcd1234wxyz5678ijklmnopqrst9012"
+          },
+          {
+            "key": "ClientSecret",
+            "value": "mnop3456qrst1234uvwx5678abcd9012"
+          },
+          {
+            "key": "OAuth2AuthzEPUrl",
+            "value": "https://test.com/authz"
+          },
+          {
+            "key": "OAuth2TokenEPUrl",
+            "value": "https://test.com/token"
+          },
+          {
+            "key": "callbackUrl",
+            "value": "https://test.com/commonauth"
+          }
+        ]
+      }
+    ]
+  },
+  "homeRealmIdentifier": "",
+  "isFederationHub": false,
+  "idpIssuerName": "",
+  "templateId": "enterprise-oidc-idp"
+}
```

Comment:
Add a new line at end

Reply:
This is update with e21d169e1176d4f00914fb72d1f624a22a0a85ea